systemtest/src/main/java/io/strimzi/systemtest/matchers/Matchers.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.matchers;
+
+import io.fabric8.kubernetes.api.model.Event;
+import io.strimzi.systemtest.k8s.Events;
+import org.hamcrest.Matcher;
+
+import java.util.List;
+
+public class Matchers {
+
+    private Matchers() {
+    }
+
+    /**
+     * Matcher to check value by key in config map
+     * @param key - key of config map
+     * @param value - expected value for the key
+     */
+    public static Matcher<String> valueOfCmEquals(String key, String value) {
+        return new CmMatcher(key, value);
+    }
+
+    /**
+     * A matcher checks that examined object has a full match of reasons for actual events.
+     * @param eventReasons - expected events for resource
+     */
+    public static Matcher<List<Event>> hasAllOfReasons(Events... eventReasons) {
+        return new HasAllOfReasons(eventReasons);
+    }
+
+    /**
+     * A matcher checks that examined object has at least one match of reasons for actual events.
+     * @param eventReasons - expected events for resource
+     */
+    public static Matcher<List<Event>> hasAnyOfReasons(Events... eventReasons) {
+        return new HasAnyOfReasons(eventReasons);
+    }
```

Comment:
I would be inclined to add a `hasNoneOfReasons` method: `return not(hasAnyOfReason(...));`. Doing it like that will make the use sites simpler.